socialNetwork/src/UserTimelineService/UserTimelineService.cpp

```diff
@@ -114,7 +120,22 @@ int main(int argc, char *argv[]) {
                            std::make_shared<TBinaryProtocolFactory>());
     LOG(info) << "Starting the user-timeline-service server with Redis Cluster support...";
     server.serve();
-  } else {
+  }
+  else if (init_redis_replica_client_pool) {
```

Comment:
are you sure it's valid variable name? maybe it should be `redis_replica_config_flag`?